### ISSUE TITLE
Decorate does nothing

### ISSUE THREAD

**Issue by MaxJohnson** [nikhilkalige/docblockr#342]
Thanks so much for this package! I use it all the time for function commenting.
Recently I tried using the decorate and doing inline or multiline decoration commands does nothing. I know it worked about a year ago, but I don't know if it's a package I installed or an Atom update that broke it.

I'm working on .js files and even if I promote the parse-enter command in the configs 
```
 'atom-text-editor:not([mini])':
  'enter': 'docblockr:parse-enter'
```
it just passes through without decorating...

**Comment by MoritzKn** [comment]
The package is not _really_ maintained anymore but what you are describing should really still work. Perhaps another keymapping is overriding the command. Try executing `docblockr:parse-enter` via the command pallet. There is also a kind of keybinding debugger...

**Comment by MaxJohnson** [comment]
Ah, thanks for the reply. Yes, I did use the keybinding debugger and the command pallet was how I was triggering. Even overriding in the config didn't help... I don't know how to make packages, but I suspect this isn't triggering correctly after some kind of update?
docblockr-worker.js:
`if(self.validate_request({scope:'comment.line.double-slash'}))`

**Comment by MoritzKn** [comment]
Can you describe you desired/expected behavior again?

**Comment by MaxJohnson** [comment]
Same as described in the documentation... single or multiline : https://github.com/nikhilkalige/docblockr#multiline-comment-decoration 

1. type in a comment
2. ctrl-shift-P to bring up the command pallet 
3. select Docblockr:Decorate (or Docblockr:Decorate Multiline)

```
// Foo bar baz<<Docblockr:Decorate>>

-- becomes

/////////////////
// Foo bar baz //
/////////////////
```
Which used to work but does not anymore. For me, at least. If I knew more about how Atom packages work maybe I could make a branch and try to fix it but I'm totally unfamiliar with that whole ecosystem.

**Comment by MoritzKn** [comment]
Ok yeah. For some reason, the `parse-enter` put me on the wrong track.
Seems like this feature is somehow broken... If you're really interested in fixing this yourself, check the atom manual it documents the internals quite well. If you are familiar with webdevlopment, it should be easy to pick up. The code of this package is kinda dated though ^^

**Comment by MoritzKn** [comment]
Seems like this was fixed by @Fluvf and now released with 0.14.0
